webapp/channels/src/components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -83,54 +84,33 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
 
     renderPostTrialCta = () => {
         const {
-            minimumSKURequiredForFeature,
             learnMoreURL,
         } = this.props;
-        if (minimumSKURequiredForFeature === LicenseSkus.Enterprise) {
-            return (
-                <div className='purchase-card'>
-                    <button
-                        className='btn btn-primary btn-lg'
-                        data-testid='featureDiscovery_primaryCallToAction'
-                        onClick={() => {
-                            this.contactSalesFunc();
-                        }}
-                    >
-                        <FormattedMessage
-                            id='admin.ldap_feature_discovery_cloud.call_to_action.primary_sales'
-                            defaultMessage='Contact sales'
-                        />
-                    </button>
-                    <ExternalLink
-                        location='feature_discovery'
-                        className='btn btn-tertiary btn-lg'
-                        href={learnMoreURL}
-                        data-testid='featureDiscovery_secondaryCallToAction'
-                    >
-                        <FormattedMessage
-                            id='admin.ldap_feature_discovery.call_to_action.secondary'
-                            defaultMessage='Learn more'
-                        />
-                    </ExternalLink>
-                </div>
-            );
-        }
```

Comment:
Removed the condition that only Enterprise feature discovery had `Contact Sales` / `Learn More` - Professional features were showing just `Purchase License` which is probably left over from self-serve purchase functionality